content-repo/extra-docs/releases/20.5.2.md

```diff
@@ -0,0 +1,271 @@
+## Cortex XSOAR Release Notes for version 20.5.2 (54345)
+##### Published on 26 May 2020
+
+### Breaking Changes
+- __TrueSTAR__  
+Changed several arguments and outputs in **trustar-get-phishing-submissions** and **trustar-get-phishing-indicators** commands. See the **Integrations** section for full details.
+
+**End Of Life Notice**: *Palo Alto Networks Cortex* Integration will reach end of life on May 31st. This is due to changes in the Cortex Data Lake move to a new version 2.0 API. Please make sure to use the *Cortex Data Lake* Integration instead.
+
+### Integrations
+
+#### 5 New Integrations
+- __IllusiveNetworks__  
+The Illusive Attack Management API allows customers to retrieve detected incidents with a forensics timeline, attack surface insights, collect forensics on-demand, and manage a variety of operations with regard to deceptive entities, deception policies, and more.
+- __Bastille Networks__  
+RF monitoring for wireless intrusion detection and policy enforcement. Visit https://www.bastille.net for details.
+- __Logz.io__  
+Fetch & remediate security incidents identified by Logz.io Cloud SIEM.
+- __Digital Guardian__  
+Use Digital Guardian Integration to fetch incidents and to programmatically add or remove entries from watchlists and component lists.
+- __ThreatMiner__  
```

Comment:
its not a new one, was just converted to directory package format
take rn from https://github.com/demisto/content/blob/master/Packs/ThreatMiner/ReleaseNotes/1_0_1.md